### PR TITLE
Finishes remaining QA items for the Inquire via Phone modal

### PR DESCRIPTION
**Pull request by yuki24**
This addresses the following QA items in https://github.com/artsy/collector-experience/issues/229#issuecomment-298762022:

 - Follow artist of the artwork after signup
 - If user signed up, when they close out of the modal, refresh the page so that they can see that they’re logged in

### CODE REVIEW

File: desktop/apps/artwork/components/inquire_via_phone/index.coffee [hunk 1]
```diff
@@ -1,15 +1,34 @@
...
+mediator = require '../../../../lib/mediator.coffee'
 
-templateMap['phone_number'] = -> require('../templates/phone_number.jade') arguments...
+templateMap['phone_number'] = -> require('./templates/phone_number.jade') arguments...
 
 module.exports = class InquireViaPhoneModalView extends AuthModalView
+  initialize: (options) ->
+    super
+    { @artistIds } = options
+    mediator.on 'modal:closed', @refreshPageIfLoggedIn
```
Comment (mzikherman) [inline desktop/apps/artwork/components/inquire_via_phone/index.coffee]: nice use of mediators for these

**Comment by mzikherman** [comment]
LGTM! I rebuilt cause of a hopefully spurious integration spec, let's see how it goes...

**Comment by yuki24** [comment]
The failure looks legit, although the helper does have [an `artistIds` function](https://github.com/artsy/force/pull/1398/files#diff-1ab845af3f781cac5727a572f255d769R25). The login failure might be related to unnecessary hooks that didn't get detached from mediators. I'll look into them further more.

**Comment by yuki24** [comment]
So the problem was that a stub was missing in `desktop/apps/artwork/components/commercial/test/template.coffee`. I also tried to run the whole build without cache in CircleCI and everything passed. this PR is good to merge now.

**Pull request merged**